.github/workflows/validator.yml

```
@@ -0,0 +1,12 @@
+name: validate-xaml
+
+on:
+  - push
```

To be consistent to other codebases.
```suggestion
on:
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]
  push:
    branches:
      - main
```